packages/nextjs/cypress/support/commands.ts

```diff
@@ -0,0 +1,105 @@
+/// <reference types="cypress" />
+// ***********************************************
+// This example commands.ts shows you how to
+// create various custom commands and overwrite
+// existing commands.
+//
+// For more comprehensive examples of custom
+// commands please read more here:
+// https://on.cypress.io/custom-commands
+// ***********************************************
+//
+//
+// -- This is a parent command --
+// Cypress.Commands.add('login', (email, password) => { ... })
+//
+//
+// -- This is a child command --
+// Cypress.Commands.add('drag', { prevSubject: 'element'}, (subject, options) => { ... })
+//
+//
+// -- This is a dual command --
+// Cypress.Commands.add('dismiss', { prevSubject: 'optional'}, (subject, options) => { ... })
+//
+//
+// -- This will overwrite an existing command --
+// Cypress.Commands.overwrite('visit', (originalFn, url, options) => { ... })
+//
+// declare global {
+//   namespace Cypress {
+//     interface Chainable {
+//       login(email: string, password: string): Chainable<void>
+//       drag(subject: string, options?: Partial<TypeOptions>): Chainable<Element>
+//       dismiss(subject: string, options?: Partial<TypeOptions>): Chainable<Element>
+//       visit(originalFn: CommandOriginalFn, url: string, options: Partial<VisitOptions>): Chainable<Element>
+//     }
+//   }
+// }
+
+Cypress.Commands.add("wakeUpHeimdall", () => {
+  const contractAddress = "0x759c0e9d7858566df8ab751026bedce462ff42df";
+  const rpcUrl = "1rpc.io/sepolia";
+
+  cy.request({
+    method: "GET",
+    url: `${Cypress.env("heimdall_url")}/${contractAddress}?rpc_url=${rpcUrl}`,
+    failOnStatusCode: false,
+    timeout: 30000,
+  }).then(response => {
+    cy.log(`Decompilation backend wake-up call completed with status: ${response.status}`);
+  });
+});
+
+Cypress.Commands.add("loadContract", (address: string, timeToWait: number) => {
+  cy.get('input[placeholder="Contract address"]').type(address);
+  cy.wait(timeToWait);
+  cy.get("button").contains("Load contract").click({ force: true });
+});
```

Comment:
I think we can remove `cy.wait` from here and should remove `{forced: true}`

[`cy.wait`](https://docs.cypress.io/api/commands/wait#__docusaurus_skipToContent_fallback): cy.wait will for sure block things until mentioned amount of time. 

But if you remove `{force: true}` from button click, [according to docs](https://arc.net/l/quote/qlltrovj) it will retry for [4sec](https://docs.cypress.io/guides/references/configuration#Timeouts) and as soon as its enabled it will interrupt and move forward (as compared to we predicting the time with `cy.wait` when button will be enabled)

Reply:
Well, I am just re-reading my code and you are totally right. Fixed it, thanks!